### DISCUSSION TITLE
testare automazioni

### DISCUSSION THREAD

**Discussion by federoco** [indomus/forum#2500] [HUB: Home Assistant]
Ciao a tutti, sto cercando di seguire le vostre guide per prendere confidenza con HA e impostare delle automazioni un po' complesse.
Ho iniziato con [questa](https://indomus.it/formazione/le-automazioni-di-home-assistant-cosa-sono-e-come-si-usano/) di spiegazione ed ho creato qualche automazione da interfaccia grafica.
Ho visto che HA si è creato il suo file automation.yaml che contiene tutte quelle che ho creato.

Sto cercando di approfondire con [questa](https://indomus.it/formazione/esercizi-di-domotica-su-home-assistant-2-definire-diversi-trigger-per-innescare-automazioni/) ma se modifico il file automation.yaml non vedo l'automazione di esempio.
Mi viene un dubbio: se parto da modalità grafica, le modifiche che faccio via SSH al file automation.yaml non vengono prese? In pratica la modalità visuale esclude quella tramite editor?
Esiste un modo per farle convivere?

**Comment by federoco** [comment]
Mi rispondo da solo: da interfaccia grafica passo direttamente in modalità YAML e lì dentro posso incollare il codice e salvare l'automazione.

**Reply by snakuzzo** [reply]
Lo puoi fare pure collegandoti in SSH e modificando il file, l'importante è che poi ricarichi le automazioni...
- o da GUI sotto DEV Tools > YAML
- o riavviando Core
